### PR TITLE
Add "supplier" pattern to function.proto

### PR DESCRIPTION
**Pull request by dsyer**
User sends a Trigger, and server sends back a stream of messages.

See #321

### CODE REVIEW

**Comment by scothis** [comment]
@dsyer how does the function-sidecar know which service to call the invoker with?

**Comment by dsyer** [comment]
It doesn't. There's no support for this in the sidecar yet. Something we need to discuss.

File: function-proto/function.proto [hunk 1]
```diff
@@ -26,6 +26,13 @@ message Message {
...
+message Trigger {
+}
+
 service MessageFunction {
   rpc Call(stream Message) returns (stream Message) {}
 }
+
+service MessageSource {
+  rpc Call(Trigger) returns (stream Message) {}
+}
```
Comment (dturanski) [inline function-proto/function.proto]: new line

Comment (dsyer) [reply function-proto/function.proto]: Fixed. Who cares though?

**Comment by dsyer** [comment]
Closing because I don't think we need it (the RPC call itself is enough of a trigger for the supplier to start).

**Pull request closed without merging**